src/Components/Footer/index.js

```diff
@@ -1,7 +1,91 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './index.scss';
 
 export default class Footer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      arrowIconOn: false,
+    };
+  }
+
+  handleArrowIcon = () => {
+    this.setState({
+      arrowIconOn: !this.state.ArrowIconOn,
+    });
```

Comment:
```suggestion
  toggleArrowIcon = () => {
    this.setState({
      arrowIconOn: !this.state.arrowIconOn,
    });
```
- toggle 하는 동작임을 더 잘 표현하는 이름이면 좋을 거 같습니다
- state는 arrowIconOn이라고 되어 있는데 현재 코드에서는 ArrowIconOn이라고, 대문자로 시작하도록 되어 있습니다. 아마 잘 동작하지 않았을 것 같은데 다시 한번 기능 확인해주세요.

Reply:
수정 완료입니다!